docs/team/hanjiyao.md

```diff
@@ -5,40 +5,42 @@ title: Jiyao's Project Portfolio Page
 
 ### Project: ContaX
 
-ContacX is for professionals who have many clients, current and prospective, who may possibly work with sensitive personal information and documents related to each of those clients. They have busy schedules with many appointments & meetings, and must be able to find available time slots in their schedules quickly.
+ContaX is an integrated solution designed for busy professionals who work with many clients and frequently interact with other people. It assists those who have packed schedules in efficiently scheduling appointments, while also being able to respond to volatile changing schedules.
 
-ContacX will allow for efficient management of a large list of contacts, allowing the mental workload of remembering people and related scheduled tasks to be offloaded. It will also provide an efficient way of efficiently organizing resources, while having minimal downtime with seamless transition from existing solutions.
+ContaX allows efficient management of a large list of contacts together with a schedule, providing an integrated solution for tracking work-related information. Built with efficiency in mind, ContaX promises a seamless transition from existing solutions for minimal downtime.
 
 Given below are my contributions to the project.
 
 * **New Feature**: Mass Operations (Chain, Range and Batch)
   * What it does: Allow user to create multiple commands together by range chain, range or batch command
   * Justification: As part of target user profile, this feature improves the usability of the application, it allows seasoned user to quickly perform different multiple commands base which suit for different situation.
-  * Highlights: `To be added soon`
-  * Credits: `To be added soon`
+
+* **New Feature**: Priority
```

Comment:
```suggestion
* **New Feature**: Appointment Priority
```
suggest to add the word `Appointment` here for clarity

Reply:
thx note changed